class Exception < Object

  def self.exception(arg0, arg1, *rest)
  end
  def to_str
  end
  def backtrace
  end
  def inspect
  end
  def message
  end
  def to_s
  end
  def exception(arg0, arg1, *rest)
  end
  def set_backtrace(arg0)
  end
end
